src/stores/game.js

```diff
@@ -225,6 +227,9 @@ export const useGameStore = defineStore('game', {
       }
     },
     opponentLeft() {
+      const opponentPnum = this.opponent.pNum;
```

Comment:
Is this used elsewhere in this better? Might be able to just inline it.

Reply:
Given its use in the template literal, I think it's easier to read this way